server/nSiteNVar_server.R

```diff
@@ -281,16 +281,18 @@ dataPrecip = reactive({
 
```

Comment:
commenting here, because it won't let me comment on binary files:
I know we .gitignored one .DS_store file, but they keep popping up. Those files and anything like .Rapp.history should be gitignored throughout the repo, wherever they arise. I think if you include those filenames in the .gitignore file as-is, they'll be ignored everywhere. You may have to copy them elsewhere, `git rm` them, and then copy them back to successfully remove them from git's history.